### PR TITLE
[MRI Violated scans] Have the minc file linked to brainbrowser (Redmine 10928)

### PR DESCRIPTION
**Pull request by gluneau**
The MRI Violated Scans now has a new column named MincFileViolated with a link that opens the scan in Brain Browser even if it is in the trashbin directory.

### CODE REVIEW

**Comment by gluneau** [comment]
Tested on IBIS production

File: modules/mri_violations/php/NDB_Menu_Filter_Form_mri_violations.class.inc [hunk 1]
```diff
@@ -171,6 +174,7 @@ class NDB_Menu_Filter_Form_mri_violations extends NDB_Menu_Filter_Form
                 c.ProjectID as Project,
                 s.SubprojectID as Subproject,
                 minc_location as MincFile,
+		        CONCAT_WS('','" . $dir_path . "','trashbin/',SUBSTRING_INDEX(minc_location, '/', -2)) as MincFileViolated,
```
Comment (driusan) [inline modules/mri_violations/php/NDB_Menu_Filter_Form_mri_violations.class.inc]: this should use the $DB->quote function

File: modules/mri_violations/php/NDB_Menu_Filter_Form_mri_violations.class.inc [hunk 2]
```diff
@@ -195,6 +199,7 @@ class NDB_Menu_Filter_Form_mri_violations extends NDB_Menu_Filter_Form
                 c.ProjectID as Project,
                 s.SubprojectID as Subproject,
                 MincFile,
+                IF(INSTR(`MincFile`, 'assembly'), IF(LEFT(`MincFile`, 8) = 'assembly',CONCAT_WS('','" . $dir_path . "',`MincFile`),`MincFile`), CONCAT_WS('','" . $dir_path . "','trashbin/',SUBSTRING_INDEX(MincFile, '/', -2))) as MincFileViolated,
```
Comment (driusan) [inline modules/mri_violations/php/NDB_Menu_Filter_Form_mri_violations.class.inc]: This line seems really complex and hard to understand. Does whatever it's doing need to be in SQL or can it be done in code?

Either way, it should at least have a comment explaining it. (dir_path should also be quoted using the db quote function)

File: modules/mri_violations/php/NDB_Menu_Filter_Form_mri_violations.class.inc (hunk 1 above)
Comment (gluneau) [inline modules/mri_violations/php/NDB_Menu_Filter_Form_mri_violations.class.inc]: Done

File: modules/mri_violations/php/NDB_Menu_Filter_Form_mri_violations.class.inc (hunk 2 above)
Comment (gluneau) [inline modules/mri_violations/php/NDB_Menu_Filter_Form_mri_violations.class.inc]: Comments added.

**Pull request merged**